### ISSUE TITLE
Flipping classroom

### ISSUE THREAD

**Issue by nickeubank** [nickeubank/poli_sci_boot_camp#4]
@alexander-dan unless you want the students' out of class time, I'd like to assign reading over tutorials for homework so we can then do coding exercises during class. No reason to waste an hour just reading already-written tutorials.

**Comment by alexander-dan** [comment]
@nickeubank just catching up -- Robin went back to work today so things have been hectic! : (

I think that's a great idea! Not sure if this is in reference to one of the emails where I wrote about there needing to be some homework to force them to do a few problems. I'm a) happy to make these pretty minimal, or b) cancel it all together (last year I just gave a bunch of problems that were entirely optional, and a couple of fun problems each day I had created, though the ones for matrices could likely be merged with some of the coding lessons).
